### PR TITLE
Add test for connect with only ssh access

### PR DESCRIPTION
**Pull request by sidpremkumar**
## Description of the change

If we have a policy with only ssh access, we should not be able to connect. This adds that test. 

## Relevant release note information

Release Notes:

## Related JIRA tickets

Relates to JIRA: CWC-1685

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [ ] No

If yes, please explain:

### CODE REVIEW

File: src/system-tests/tests/suites/ssh.ts [hunk 1]
```diff
@@ -106,6 +108,31 @@ export const sshSuite = () => {
             }, 60 * 1000);
         });
 
+        ssmTestTargetsToRun.forEach(async (testTarget: TestTarget) => {
+            it(`${testTarget.sshCaseId}: connect test with only tunnel policy - ${testTarget.awsRegion} - ${testTarget.installType} - ${testTarget.dropletImage}`, async () => {
```
Comment (chad-cwc) [inline src/system-tests/tests/suites/ssh.ts:112]: Would it be helpful if this description says something like 'connect should fail with tunnel-only policy'?

Comment (sidpremkumar) [reply src/system-tests/tests/suites/ssh.ts]: https://github.com/bastionzero/zli/pull/373/commits/8d9f758207dce1751e217351936c9a70d244395c

Comment (chad-cwc) [reply src/system-tests/tests/suites/ssh.ts]: Sorry to be nitpicky here, but your new text "connect test should fail with only tunnel policy" reads like the test should fail. We know it's a test so the word test doesn't need to be in there anyway.

Comment (sidpremkumar) [reply src/system-tests/tests/suites/ssh.ts]: https://github.com/bastionzero/zli/pull/373/commits/c7dbdecc70ab89b32e48356a894f3c868a63f50b

**Review by chad-cwc — APPROVED** [review]

**Pull request merged**